backend/db.js

```diff
@@ -52,6 +52,22 @@ export const isLatestDatabase = async () => {
   return localPackage.version === remotePackage.version
 }
 
+/**
+ * Get local database version.
+ * @async
+ * @returns {string} Local database verison.
+ */
+export const getDbVersion = async () => {
+  // Read package.json database semver and database package file
+  const [ , { version } ] = await Promise.all( [
+    manifest( databasePackage ),
```

Comment:
What does this do? Read the database package's json? Why is there a readJSON below it?

Reply:
It reads the DB version from node_modules

Reply:
What does the line below do/why are there two lines from reading database package? Did you source the code from Desktop?

Reply:
Oh, I think one must've been for the npm DB version

Reply:
I fixed it now.